### PR TITLE
New block bugfix

### PR DESCRIPTION
**Pull request by brianosaurus**
NewBlock events now properly report amount synced etc...

### CODE REVIEW

File: packages/augur-sdk/src/state/Controller.ts [hunk 1]
```diff
@@ -44,9 +44,16 @@ export class Controller {
   }
 
   private notifyNewBlockEvent = async (): Promise<void> => {
-    const lowestBlock = await (await this.db).syncStatus.getLowestSyncingBlockForAllDBs();
+    let lowestBlock = await (await this.db).syncStatus.getLowestSyncingBlockForAllDBs();
+    if (lowestBlock == -1) {
+
```
Comment (robert-e-davidson3) [inline packages/augur-sdk/src/state/Controller.ts]: What's supposed to go here?

File: packages/augur-sdk/src/state/db/DB.ts [hunk 2]
```diff
@@ -167,36 +167,45 @@ export class DB {
...
         let dbName = this.getDatabaseName(userSpecificEvent.name, trackedUser);
+        try {
+          dbSyncPromises.push(
+            this.syncableDatabases[dbName].sync(
+              augur,
+              chunkSize,
+              blockstreamDelay,
+              highestAvailableBlockNumber
+            ));
+        } catch (e) {
```
Comment (robert-e-davidson3) [inline packages/augur-sdk/src/state/db/DB.ts:178]: We should at least be logging the errors we're eating.

File: packages/augur-sdk/src/state/db/DB.ts [hunk 3]
```diff
@@ -167,36 +167,45 @@ export class DB {
...
+        }
+      }
+    }
+
+    for (let genericEventName of this.genericEventNames) {
```
Comment (robert-e-davidson3) [inline packages/augur-sdk/src/state/db/DB.ts]: I _think_ this can be a `const`.

File: packages/augur-sdk/src/state/db/DB.ts [hunk 4]
```diff
@@ -167,36 +167,45 @@ export class DB {
...
+      let dbName = this.getDatabaseName(genericEventName);
+      try {
         dbSyncPromises.push(
           this.syncableDatabases[dbName].sync(
             augur,
             chunkSize,
             blockstreamDelay,
             highestAvailableBlockNumber
           ));
+      } catch (e) {
```
Comment (robert-e-davidson3) [inline packages/augur-sdk/src/state/db/DB.ts]: log error?

File: packages/augur-sdk/src/state/db/DB.ts (hunk 2 above)
Comment (brianosaurus) [reply packages/augur-sdk/src/state/db/DB.ts:178]: kk, I'll change that.

File: packages/augur-sdk/src/state/db/DB.ts (hunk 3 above)
Comment (brianosaurus) [reply packages/augur-sdk/src/state/db/DB.ts]: Fixed all of the let's I could.

File: packages/augur-sdk/src/state/Controller.ts (hunk 1 above)
Comment (brianosaurus) [reply packages/augur-sdk/src/state/Controller.ts]: oops, nice catch. Fixed.

**Review by robert-e-davidson3 — APPROVED** [review]

**Review by bthaile — APPROVED** [review]
lgtm, I'm getting correct values for latest block and current block

**Pull request merged**